src/context/yaml/handlers/actions.ts

```diff
@@ -22,6 +22,7 @@ function parseCode(context: YAMLContext, code: string) {
 
 async function parse(context: YAMLContext): Promise<ParsedActions> {
   // Load the script file for each action
+  //@ts-ignore TODO: understand if empty array is intentionally being returned
```

Comment:
This is an example of a compilation error that may be the source of a bug. In this case, we'd expect the return type to be an object with the key `actions`.